### DISCUSSION TITLE
Quick Capture - Record quick ideas, notes, time logging, etc.

### DISCUSSION THREAD

**Discussion by GitMurf** [SilentVoid13/Templater#164] [Templates Showcase]
https://gist.github.com/GitMurf/817a6c9e73e1d1e312fc1a1735edb8d6

Obsidian Quick Capture using templater. Activate from anywhere in your vault to record quick ideas, notes, time logging, etc. The Quick Capture file does NOT need to be open.

QUICK DEMO: https://user-images.githubusercontent.com/64155612/115743581-a66afe00-a346-11eb-9c38-f25b6ebc4504.mp4

**Comment by GitMurf** [comment]
v1.4: Added ability to turn option on to nest each new quick capture entry under a header with the Daily Notes Page link [[DNP]] for each day. Just change to: let bAddHeader = true;

![image](https://user-images.githubusercontent.com/64155612/115770377-7846e780-a361-11eb-8cd9-a1e9f9c2ebbe.png)

**Comment by GitMurf** [comment]
To create a "task" just start the quick capture text with a semicolon ";"

![image](https://user-images.githubusercontent.com/64155612/116971301-332e8980-ac6e-11eb-9912-03fc95c6ae8e.png)

**Comment by iroQuai** [comment]
This looks wonderful! Will try it out soon :)

**Comment by davejunkie** [comment]
I am new to templater but trying to get this to work. I am getting an error and it's not working. I have created the Quick Capture note but not working. 
I have created a template and paste the quick capture code in, it is in my templates folder.
Would anyone be able to give me some help?

Templater Error: Template parsing error, aborting. Check console for more informations
![quickcapture templater error 021-05-16 180300](https://user-images.githubusercontent.com/17958980/118425060-3c135800-b67d-11eb-97e0-cacf4b9cc642.png)

**Reply by GitMurf** [reply]
@davejunkie did you follow my video on running your first Templater script here? https://github.com/SilentVoid13/Templater/discussions/187

I suspect it may be how you are pasting the code in. You need to make sure to use ctrl + shift + v for pasting as plain text.

**Comment by LoneExile** [comment]
Thanks for the template man, I love it.

**Reply by LoneExile** [reply]
My current note got enter by 8 lines when I use it,
need to cmd+z(undo) every time I use the template.
